Sun Aug  9 15:00:01 CDT 2020
Traceback (most recent call last):
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 170, in <module>
    main()
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 143, in main
    oldest = newest - date.timedelta(days=7)
NameError: name 'date' is not defined
Sun Aug 16 15:00:01 CDT 2020
Traceback (most recent call last):
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 170, in <module>
    main()
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 143, in main
    oldest = newest - date.timedelta(days=7)
NameError: name 'date' is not defined
Fri Aug 28 13:25:10 CDT 2020
Traceback (most recent call last):
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 170, in <module>
    main()
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 143, in main
    oldest = newest - date.timedelta(days=7)
NameError: name 'date' is not defined
Sun Aug 30 15:00:02 CDT 2020
Traceback (most recent call last):
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 170, in <module>
    main()
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 143, in main
    oldest = newest - date.timedelta(days=7)
NameError: name 'date' is not defined
Thu Sep  3 14:57:48 CDT 2020
Traceback (most recent call last):
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 170, in <module>
    main()
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 143, in main
    oldest = newest - date.timedelta(days=7)
NameError: name 'date' is not defined
Thu Sep  3 15:01:47 CDT 2020
Traceback (most recent call last):
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 170, in <module>
    main()
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 143, in main
    oldest = newest - datetime.date.timedelta(days=7)
AttributeError: type object 'datetime.date' has no attribute 'timedelta'
Thu Sep  3 15:05:44 CDT 2020
Traceback (most recent call last):
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 170, in <module>
    main()
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 167, in main
    makeCSV(logFolder,csvFolder,webPath,header,newest,oldest,csvLogFilePath)
  File "/mnt/c/Users/Image/Documents/WorkflowScriptsWS2/WeeklyCSV.py", line 52, in makeCSV
    for file in os.listdir(logFolder):
OSError: [Errno 22] Invalid argument: '/mnt/e/CFLA-LSU-Station2/LSUCollections/Logs/'
Thu Sep  3 15:10:25 CDT 2020
Sun Sep  6 15:00:02 CDT 2020
Sun Sep 13 15:00:02 CDT 2020
Sun Sep 20 15:00:02 CDT 2020
Sun Sep 27 15:00:02 CDT 2020
Sun Oct  4 15:00:02 CDT 2020
Sun Oct 11 15:00:02 CDT 2020
Sun Oct 18 15:00:21 CDT 2020
Sun Oct 25 15:00:02 CDT 2020
